frontend/rolecall/src/app/api/cast_api.service.ts

```diff
@@ -417,7 +417,7 @@ async requestCastSet(cast: Cast): Promise<HttpResponse<any>> {
     return undefined;
   }
 
-  static castUUIDFromRaw(id: number) {
+  static uuidFromRaw(id: number) {
```

Comment:
This util doesn't really have much to do with UUID or the CastAPI class. If it's really needed, maybe it can be just a generic global util
```
export function numToString(num: number) {
  return String(num);
}
```

Though I wonder how useful that is, as it's simpler to just write 
`String(this.castAPI.lastSavedCastId)`
instead of 
`numToString(this.castAPI.lastSavedCastId`
`CastApi.uuidFromRaw(this.castAPI.lastSavedCastId)`

Reply:
Removed.